common/api/appui-react.api.md

```diff
@@ -2995,10 +3062,10 @@ export class MenuItemHelpers {
     // (undocumented)
     static createMenuItemNodes(itemList: MenuItem[]): React_2.ReactNode[];
     // (undocumented)
-    static createMenuItems(itemPropsList: MenuItemProps[], onSelection?: () => void): MenuItem[];
+    static createMenuItems(itemPropsList: CursorMenuItemProps[], onSelection?: () => void): MenuItem[];
 }
 
-// @public
+// @public @deprecated
```

Comment:
Can we get away w/o deprecation and introduction of `CursorMenuItemProps`? Are `CursorMenuItemProps` and `AbstractMenuItemProps` identical?

Reply:
No, they're not the same. The new `CursorMenuItemProps` extends `ItemProps` from `appui-react` rather than `CommonItemProps` from `appui-abstract`. Even without that, we'd run into issues with {String/Boolean}ConditionalValue classes not being interchangeable (private member class comparison).

Reply:
Btw, can/should we do the same trick we did with enums for ConditionalValue classes?

Reply:
Good idea, let me see.

Reply:
Looks like we can. Changed.